CMakeLists.txt

```diff
@@ -51,10 +51,14 @@ else()
     )
 endif()
 
```

Comment:
Can you add a cmake option? `option(DISABLE_FILL_FORM_SHORTCUTS "Build without form fill support" OFF)`